src/libraries/Common/src/Interop/Windows/Kernel32/Interop.GetSystemInfo.cs

```diff
@@ -7,7 +7,7 @@ internal static partial class Interop
 {
     internal static partial class Kernel32
     {
-        [DllImport(Libraries.Kernel32)]
-        internal static extern void GetSystemInfo(out SYSTEM_INFO lpSystemInfo);
+        [GeneratedDllImport(Libraries.Kernel32)]
+        internal static partial void GetSystemInfo(out SYSTEM_INFO lpSystemInfo);
```

Comment:
Quick sanity check here. I assume that `SYSTEM_INFO` is blittable, so we could technically change this to `internal static extern unsafe void GetSystemInfo(SYSTEM_INFO* lpSystemInfo);` and make a fixed statement at the invocation site right? I don't know how big this generated code is but it seems like we might want to consider that. I would argue we have a different level of expectation for P/Invokes in the BCL and if we can avoid size it might be worth skipping the added function.

Reply:
Did this change the size impact at all?

Reply:
Yes - I just updated the description. Before the change, it was:
| Library | Original (B) | Converted (B) | + (B) | + (%) |
| ------- | -----------: | ------------: | ----: | ----: |
| windows | 12288 | 12800 | 512 | 4.17 |
| windows R2R | 18944 | 20480 | 1536 | 8.11 |